docs/iot/tutorials/gpio-input.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Use GPIO for binary input
+description: Learn how to use GPIO for binary input with the .NET IoT Libraries.
+author: camsoper
+ms.author: casoper
+ms.date: 10/04/2022
+ms.topic: tutorial
+ms.prod: dotnet
+recommendations: false
+---
+
+# Use GPIO for binary input
+
+General-purpose I/O (GPIO) pins can be configured to receive electrical signals as input. At its most basic level, this is useful for scenarios that detect the opening/closing of a circuit. Such circuits might include push buttons, toggle switches, reed switches, pressure switches, and other devices that represent binary (on/off) values by completing a circuit.
+
+In this tutorial, you'll use .NET and your Raspberry Pi's GPIO pins to detect the opening and closing of a circuit.
+
+## Prerequisites
+
+- [!INCLUDE [prereq-rpi](../includes/prereq-rpi.md)]
+- Jumper wires
+- Breadboard (optional)
+- Raspberry Pi GPIO breakout board (optional)
+- [!INCLUDE [tutorial-prereq-dotnet](../includes/tutorial-prereq-dotnet.md)]
+
+[!INCLUDE [ensure-ssh](../includes/ensure-ssh.md)]
+
+## Prepare the hardware
+
+Use the hardware components to build the circuit as depicted in the following diagram:
+
+:::image type="content" source="../media/rpi-jumper-21-gnd-thumb.png" alt-text="A diagram showing a circuit that connects a ground pin to pin 21." lightbox="../media/rpi-jumper-21-gnd-thumb.png":::
+
+The image above depicts a direct connection between a ground pin and pin 21.
```

Comment:
Is it ok to assume people would know which pin is pin 21? I'm not sure how many people are used to the Gpio numbering, there are also different numbering schemes to be aware of, like the board pin numbering or the logical GPIO one.